benches/bench.rs

```diff
@@ -207,6 +207,21 @@ fn build_cloneable(b: &mut Bencher) {
     });
 }
 
+fn access_column(b: &mut Bencher) {
+    let mut world = World::new();
+    let _enta = world.spawn((Position(0.0), Velocity(0.0)));
+    let _entb = world.spawn((true, 12));
+    let entc = world.spawn((Position(3.0),));
+    let _entd = world.spawn((13, true, 4.0));
+    let column = world.column::<Position>();
+
+    b.iter(|| {
+        for i in 0..100 {
+            let comp = column.get(entc).unwrap();
```

Comment:
We need to ensure this isn't getting optimized out entirely. `bencher::black_box` should hopefully do the trick.